app/assets/stylesheets/common/select-kit/toolbar-popup-menu-options.scss

```diff
@@ -39,3 +35,18 @@
     }
   }
 }
+
+// ensure popupmenu options has scroll behavior on limited screen space
+#reply-control {
+  &.composer-action-createTopic {
+    .toolbar-popup-menu-options .select-kit-collection {
+      max-height: calc(var(--composer-height) - 230px);
+    }
+  }
+
+  &.composer-action-reply {
+    .toolbar-popup-menu-options .select-kit-collection {
+      max-height: calc(var(--composer-height) - 145px);
```

Comment:
Hmm... so, I think this approach (that I recommended, so my mistake) is a bit tricky, especially with the reply composer. If I have a tall enough window but a small reply composer panel, the dropdown is needlessly small. 

It gives this when I pull the branch locally: 
<img width="750" alt="image" src="https://user-images.githubusercontent.com/368961/187227929-dd662052-ba3f-4435-ad8d-6a7aadb33dd7.png">

whereas in current core it shows like this: 

<img width="750" alt="image" src="https://user-images.githubusercontent.com/368961/187228286-49f04a28-2355-4dc8-a4a5-6dfbd1271d2f.png">

So... following the `composer-height` is not necessarily a great idea for all window sizes... I guess we have two options: 

- go back to the simple calculation based on `vh` but adjust it for mobile
- or use a `max()` function that picks one of viewport height or composer height, something like `max-height: Max(40vh, calc(var(--composer-height) - 130px));`

I'm OK with either option. Maybe just the simple option is best long-term, easier to manage in the long run, but I'll leave the decision up to you.